binding/format/protobuf/v2/pb/cloudevent_grpc.pb.go

```diff
@@ -0,0 +1,175 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
+// versions:
+// - protoc-gen-go-grpc v1.3.0
+// - protoc             v3.12.4
+// source: cloudevent.proto
+
+package pb
+
+import (
+	context "context"
+	empty "github.com/golang/protobuf/ptypes/empty"
+	grpc "google.golang.org/grpc"
+	codes "google.golang.org/grpc/codes"
+	status "google.golang.org/grpc/status"
+)
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the grpc package it is being compiled against.
+// Requires gRPC-Go v1.32.0 or later.
+const _ = grpc.SupportPackageIsVersion7
+
+const (
+	CloudEventService_Publish_FullMethodName   = "/pb.CloudEventService/Publish"
+	CloudEventService_Subscribe_FullMethodName = "/pb.CloudEventService/Subscribe"
+)
+
+// CloudEventServiceClient is the client API for CloudEventService service.
+//
+// For semantics around ctx use and closing/ending streaming RPCs, please refer to https://pkg.go.dev/google.golang.org/grpc/?tab=doc#ClientConn.NewStream.
+type CloudEventServiceClient interface {
```

Comment:
The Publish-subscribe model has the following characteristics:
1. The publisher sends events without specifically targeting any recipient. These messages are broadcasted to all interested subscribers.
2. Subscribers express interest in certain types of messages and receive notifications or updates when messages of interest are published. This allows for a loosely coupled communication model where publishers and subscribers are decoupled from each other.

In my opinion, the grpc implementation seems more suitable for the request-response model. 
The requester initiates communication with a specific recipient, and the recipient responds directly to the requester's query or request. So I suggest naming the interface `Publish/Subscribe` to `Request/Response`